armotypes/portaltypesutils.go

```diff
@@ -274,3 +275,29 @@ func (p *PortalBase) GetUpdatedTime() *time.Time {
 	}
 	return &updatedTime
 }
+
+func (p *PortalCustomer) GetCustomerStatus(now int64) CustomerStatus {
+	if p.ActiveSubscription == nil || p.ActiveSubscription.LicenseType == LicenseTypeFree {
```

Comment:
Please add commenting to explain the logic of concluding each type of customerStatus.